src/models/transfers.js

```diff
@@ -207,31 +208,83 @@ function * processStateTransitions (tr, transfer) {
       yield accountBalances.applyDebits()
       updateState(transfer, 'prepared')
     }
+
+    // If the transfer is prepared and has no execution condition, execute immediately
```

Comment:
Good point. I updated this after naming the method. I think there is some larger refactoring that can be done here, but for now I'll just change the method name.